man/nvim.1

```diff
@@ -0,0 +1,375 @@
+See
+.Ic :help recovery .
+.It Fl u Ar nvimrc
+Use
+.Ar nvimrc
+instead of the default of
+.Pa ~/.nvimrc .
+If
+.Ar nvimrc
+is
+.Sy NONE ,
+do not read any initialization files or environment variables.
+See
+.Ic :help initialization .
+.It Fl i Ar nviminfo
+Use
+.Ar nviminfo
+instead of the default of
+.Pa ~/.nviminfo .
+If
+.Ar nviminfo
+is
+.Sy NONE ,
+do not read or write an nviminfo file.
+See
+.Ic :help viminfo .
+.It Fl -noplugin
+Skip loading plugins; implied by
+.Li Sy -u NONE .
+.It Fl o Ns Op Ar N
+Open
+.Ar N
+windows stacked horizontally.
+If
+.Ar N
+is omitted, open one window for each file.
+.It Fl O Ns Op Ar N
+Open
+.Ar N
+windows stacked vertically.
+If
+.Ar N
+is omitted, open one window for each file.
+.It Fl p Ns Op Ar N
+Open
+.Ar N
+tab pages.
+If
+.Ar N
+is omitted, open one tab page for each file.
+.It Cm + Ns Op Ar linenum
+For the first file, position the cursor on line
+.Ar linenum .
+If
+.Ar linenum
+is omitted, position the cursor on the last line of the file.
+Note that the command-line argument
+.Sy +5
+is equivalent to
+.Sy :5
+inside
+.Nm .
+.It Cm +/ Ns Op Ar pattern
+For the first file, position the cursor on the first occurrence of
+.Ar pattern .
+If
+.Ar pattern
+is omitted, the most recently used search pattern is used (if there is one).
+Note that the command-line argument
+.Sy /foo
+is equivalent to
+.Sy /foo
+inside
+.Nm .
+See
+.Ic :help search-pattern .
+.It Fl -cmd Ar command
+Execute
+.Ar command
+before processing any nvimrc.
+Up to 10 instances of these can be used, independently from instances of
+.Fl c .
+.It Fl c Ar command
+Execute
+.Ar command
+after reading the first file.
+Up to 10 instances of
+.Fl c
+or
+.Cm +
+can be used.
+.It Fl S Op Ar session
+Source
+.Ar session
+after the first file argument has been read; equivalent to
+.Sy -c \(dqsource session\(dq .
+.Ar session
+cannot start with a hyphen.
+If
+.Ar session
+is omitted, then
+.Pa Session.vim
+is used (only works if
+.Fl S
+is the last argument specified).
+.It Fl s Ar scriptin
+Read normal mode commands from
+.Ar scriptin .
+The same can be done with the command
+.Ic :source! scriptin .
+If the end of the file is reached before
+.Nm
+exits, further characters are read from the keyboard.
+.It Fl w Ar scriptout
+Append all typed characters to
+.Ar scriptout .
+Can be used for creating a script to be used with
+.Fl s
+or
+.Ic :source! .
+.It Fl W Ar scriptout
+Like
+.Fl w ,
+but truncate
+.Ar scriptout .
+.It Fl -startuptime Ar file
+During startup, append timing messages to
+.Ar file .
+Can be used to diagnose slow startup times.
+.It Fl -api-info
+Dump API metadata serialized to msgpack and exit.
+.It Fl -embed
+Use standard input and standard output as a msgpack-rpc channel; implies
+.Fl -headless .
+.It Fl -headless
+Do not start a user interface.
+.It Fl h , -help
+Print usage information and exit.
+.It Fl v , -version
+Print version information and exit.
+.El
+.Sh ENVIRONMENT
+.Bl -tag -width VIMRUNTIME
+.It Ev VIMINIT
```

Comment:
Some of these env variable descriptions aren't very good, probably because I don't really know what differentiates some of them. Does anyone have any improvements?